### PR TITLE
Fix Android read-only error 

### LINKED ISSUE #1: “Attempting to change value of a readonly property” in createPropType.js
The following line throws the error in the subject on Adroid (with react-native).
`Object.defineProperty(propType, 'name', { value: type });`

Note that that line does not cause any issues on iOS. I [resolved it](https://github.com/ixje/react-extra-prop-types/commit/62f8a27e1d2ed815b3fffff4d155bc25f1b7f18d) by using assignment instead of definition. I'm aware there are some differences for mutability and for prototype, but for this lib I'm not really worried about that. The test-cases all pass with this simple fix. 

I currently don't know of a better way to fix this quickly, but I leave that up to you to decide if this is an acceptable solution. I can PR if you want (after removing the commented line).

### PR DESCRIPTION
**Pull request by ixje**
Fixes #1.

### CODE REVIEW

File: lib/createPropType.js [hunk 1]
```diff
@@ -11,8 +11,11 @@ function createPropType (type, validator) {
       return getPropTypeError(type, propName, componentName);
     }
   }
-  Object.defineProperty(propType, 'name', { value: type });
-
+  try {
+    Object.defineProperty(propType, 'name', { value: type })
+  } catch (e) {
+    // https://github.com/benwiley4000/react-extra-prop-types/issues/1
+  }
```
Comment (benwiley4000) [inline lib/createPropType.js]: Total nit, but could we get a semicolon, and a newline after the `catch`? Thanks! 🙂

**Review by benwiley4000 — CHANGES_REQUESTED** [review]
Looks great - thanks for catching this. Just a couple tiny style/formatting changes.

Also, looks like you're not committing with your GitHub email, so the commits won't be attributed to your account - in case that matters to you! Feel free to force push if you want to [update those commits with new info](https://gist.github.com/trey/9588090).

**Comment by ixje** [comment]
I hope that will do it :)

**Comment by benwiley4000** [comment]
Haha sorry, should have been more obvious; the semicolon goes at the end of the `Object.defineProperty(...)` call, not the catch 😬

**Comment by ixje** [comment]
Hehe no worries, there are so many coding styles that I didn't even think about whether it made sense or not.

**Review by benwiley4000 — APPROVED** [review]

**Comment by benwiley4000** [comment]
Thanks! 😄

**Comment by benwiley4000** [comment]
Published to npm, v0.3.1. 🎉

**Pull request merged**